AirCasting/SearchAndFollow/SearchLocation/SearchPickerService.swift

```diff
@@ -3,15 +3,20 @@
 import Foundation
 import GooglePlaces
 import SwiftUI
+import Resolver
 
 class SearchPickerService: PlacePickerService {
-    @Binding private var address: String
+    @InjectedObject private var tracker: LocationTracker
+    @Binding private var addressName: String
+    @Binding private var addressLocation: CLLocationCoordinate2D
     
-    init(address: Binding<String>) {
-        self._address = .init(projectedValue: address)
+    init(addressName: Binding<String>, addressLocation: Binding<CLLocationCoordinate2D>) {
+        self._addressName = .init(projectedValue: addressName)
+        self._addressLocation = .init(projectedValue: addressLocation)
     }
     
     func didComplete(using place: GMSPlace) {
-        address = place.name ?? ""
+        addressName = place.name ?? ""
+        addressLocation = place.coordinate
```

Comment:
what would happen when if the user had a poor connection and the time between `addressName` update and getting the `addressLocation` is long? Will the user be able to press next in between or smth and get to the wrong state?

Reply:
I need to say it was done here but now this change is gone :/